Scripts/Editor/NodeEditorAction.cs

```diff
@@ -483,7 +486,7 @@ public enum NodeActivity { Idle, HoldNode, DragNode, HoldGrid, DragGrid }
                 if (draggedOutputTarget != null) gridPoints.Add(portConnectionPoints[draggedOutputTarget].center);
                 else gridPoints.Add(WindowToGridPosition(Event.current.mousePosition));
 
-                DrawNoodle(col, gridPoints);
+                DrawNoodle(g, gridPoints, true);

```

Comment:
Use full name `gradient` instead of `g`, as `g` doesn't convey that it is a gradient.